### PR TITLE
rework some of the language of the website

### PR DESCRIPTION
**Pull request by fjy**

### CODE REVIEW

File: druid.md [hunk 1]
```diff
@@ -24,19 +24,17 @@ provides a framework for plugging in new functionality. Druid also supports
...
+**Real-time Ingestion** Typical analytics databases ingest data via batches.
 Ingesting an event at a time is often accompanied with transactional locks and
-other overhead that slows down the ingestion rate. Druid's real-time nodes
-employ lock-free ingestion of append-heavy data sets to allow for simultaneous
-ingestion and querying of 10,000+ events per second. Simply put, the latency
-between when an event happens and when it is visible is limited only by how
-quickly the event can be delivered to Druid.
+other overhead that slows down the ingestion rate. Druid employs lock-free
+ingestion of append-heavy data sets to allow for simultaneous ingestion and
+querying of 10,000+ events per second. Simply put, the latency between when an
```
Comment (gianm) [inline druid.md]: Per second per node? Per second per core? Otherwise this sounds like Druid can only handle 10,000 events per second

File: index.html [hunk 2]
```diff
@@ -12,7 +12,7 @@
   <div class="row">
     <div class="col-md-8 col-md-offset-2 text-center">
     <span class="druid-bootlogo"></span>
-    <p class="lead">Interactive Analytics at Scale</p>
+    <p class="lead">Open source streaming analytics data store</p>
```
Comment (gianm) [inline index.html]: Why these changes?

File: druid.md [hunk 3]
```diff
@@ -24,19 +24,17 @@ provides a framework for plugging in new functionality. Druid also supports
...
 calculations.
 
-**Interactive Queries** Druid’s query latency is optimized by reading and
-scanning only exactly what is needed. Aggregate and filter on data without
-sitting around waiting for results. Druid is ideal for powering analytic
-dashboards. Druid is a column-oriented data store and supports ad-hoc,
-  multi-dimensional filtering.
+**Sub-second Queries** Druid’s column orientation and inverted indexes enable
+scanning exactly what is needed for a query. Aggregate and filter on data in
+milliseconds. Druid is ideal for powering analytic dashboards.
```
Comment (gianm) [inline druid.md]: Why remove ad-hoc multi-dimensional filtering? Many other things out there don't do it well.

File: druid.md [hunk 4]
```diff
@@ -6,7 +6,7 @@ sectionid: druid
 
 Druid is an open-source analytics data store designed for business intelligence
 ([OLAP](http://en.wikipedia.org/wiki/Online_analytical_processing))
-queries on timeseries data. Druid provides low latency (real-time) data
+queries on event data. Druid provides low latency (real-time) data
```
Comment (gianm) [inline druid.md]: Generally on board with replacing "timeseries data" by "event data". No comment here other than agreement :P

File: druid.md [hunk 5]
```diff
@@ -54,12 +53,13 @@ Large Druid production clusters have scaled to following:
...
 - 30+ trillion events
 - Hundreds of queries per second for applications used by thousands of users
+- Tens of thousands of cores
 
 ## Is Druid Right for Me?
 
-Organizations have deployed Druid to analyze ad-tech, dev ops, network,
-activity stream (website), finance, and sensor data. In particular, Druid is a
-good fit if you have the following requirements:
+Organizations have deployed Druid to analyze ad-tech, dev ops, network traffic,
```
Comment (xvrl) [inline druid.md]: "dev-ops" instead of "dev ops"

File: druid.md [hunk 6]
```diff
@@ -24,19 +24,18 @@ provides a framework for plugging in new functionality. Druid also supports
...
 
-**Interactive Queries** Druid’s query latency is optimized by reading and
-scanning only exactly what is needed. Aggregate and filter on data without
-sitting around waiting for results. Druid is ideal for powering analytic
-dashboards. Druid is a column-oriented data store and supports ad-hoc,
-  multi-dimensional filtering.
+**Sub-second Queries** Druid’s column orientation and inverted indexes enables
+complex multi-dimensional filtering and scanning exactly what is needed for a
+query. Aggregate and filter on data in milliseconds. Druid is ideal for
+powering analytic dashboards.
```
Comment (xvrl) [inline druid.md]: add interactive?

File: index.html [hunk 7]
```diff
@@ -40,7 +40,7 @@
         <span class="fa fa-database fa-4x"></span><br>
         <h2>Scalable</h2>
       </a></p>
-      <p class="text-justify">Scalable and available; consistent and predictable performance in multi-tenant environments. Scale up to increase computation power, scale down to save on costs.</p>
+      <p class="text-justify">Scalable and available; consistent and predictable performance in multi-tenant environments. Existing clusters have scaled to petabytes of data and trillions of events.</p>
```
Comment (xvrl) [inline index.html]: should we add ingestion performance here? e.g. 1M events/s?
